x/stake/client/cli/query.go

```diff
@@ -93,6 +52,41 @@ func GetCmdQueryCandidate(storeName string, cdc *wire.Codec) *cobra.Command {
 	return cmd
 }
 
+// get the command to query a candidate
+func GetCmdQueryCandidates(storeName string, cdc *wire.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "candidates",
```

Comment:
What order are candidates returned in?

Maybe accept a limit (and offset) of the number of candidates to return?

Reply:
Candidates are returned in the stored order (by candidate owner address) yeah we could build in a limit, just one more thing to parse for in the path though, which is kind of confusing, it would probably need to use a different symbol such as a dash, ex. `subspace-100` not sure I love this idea. I think it's maybe a bit too much complexity that we don't need in the query, you can manually filter after you get the results - also not concerned about limiting so much as per this issue: https://github.com/cosmos/cosmos-sdk/issues/975

Reply:
Fair enough, not a priority, agreed we want to avoid ugliness/complexity.